Lib/Magnet/HotKey.swift

```diff
@@ -41,6 +42,7 @@ public final class HotKey: Equatable {
 
     // MARK: - Initialize
     public init(identifier: String, keyCombo: KeyCombo, target: AnyObject, action: Selector, actionQueue: ActionQueue = .main) {
+        super.init()
```

Comment:
Please call `super.init()` method at the end of initializer.

Reply:
Fixed with a00a0ed